app/views/collections/_collect.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if collect = current_user.collection_for(story) %>
```

Comment:
if this is something that they collected?

I wouldn't do these kind of condtionals inside view it should be in a controller.